Source/WebKit/Shared/libwpe/WebEventFactory.cpp

```diff
@@ -157,6 +157,37 @@ static inline short pressedMouseButtons(uint32_t modifiers)
     return buttons;
 }
 
+static inline uint32_t getDoubleClickTime()
+{
+    return 500; // Milliseconds.
+}
```

Comment:
Why do we need a function for this? it can be a static const, and the pixelThreshold too.